lib/auth/trustedcluster.go

```diff
@@ -388,19 +388,21 @@ func (a *Server) GetRemoteCluster(clusterName string) (types.RemoteCluster, erro
 	return remoteCluster, trace.Wrap(err)
 }
 
-func (a *Server) updateRemoteClusterStatus(ctx context.Context, netConfig types.ClusterNetworkingConfig, remoteCluster types.RemoteCluster) error {
+// updateRemoteClusterStatus determines current connection status of remoteCluster and writes it to the backend
+// if there are changes. Returns true if backend was updated.
```

Comment:
Could you add a brief explanation on what happens when `(false, nil)` is returned? Naively I would expect that a nil err means a success.

Reply:
Good point. Updated the comment in https://github.com/gravitational/teleport/pull/22455/commits/9c69ca171005041eed0a31493120fc3b8baad341